pages/searching/csv_export.rst

```diff
@@ -1,17 +1,55 @@
 Export results as CSV
 ^^^^^^^^^^^^^^^^^^^^^
-It is also possible to export the results of your search as a CSV document. To do so, click on the three dots on the right side of the search bar and select the *Export as CSV* option.
+It is possible to export the results of your search as a CSV document. To do so, click on the three dots on the right side of the search bar and select the *Export to CSV* option.
 
 .. image:: /images/searching/csv_export.png
    :align: center
 
-This will open a modal where you can choose the fields the CSV document should contain. You can also optionally specify a stream to filter the messages.
-Clicking on *Downlaod* will create the file based on the specified options and start the download.
+This will open a modal where you can choose the fields the CSV document should contain. The initial fields are based on the message table configured for the search page.
+It is also possible to define a limit. Because messages are loaded in chunks, all chunks up to the one where the limit is reached will be retrieved.
```

Comment:
Because messages are loaded in chunks of fixed size, and because the final chunk will rarely end exactly at that fixed size, it is likely that the total number of messages exported will sligtly exceed the number defined by the user.